src/tooltip/test/tooltip.spec.js

```diff
@@ -249,10 +249,11 @@ describe('tooltip', function() {
       scope.$digest();
     }));
 
-    it('should open after timeout', inject(function ($timeout) {
+    it('should open after timeout and set the close delay to the default', inject(function ($timeout) {
 
       elm.trigger('mouseenter');
       expect(tooltipScope.isOpen).toBe(false);
```

Comment:
Strictly speaking, this doesn't really test that the `tooltip` is closed, only that its scope's `isOpen` is `false`. This **should** be the same, in principle, but only **assuming** it works correctly. 

Would a direct testing of opening/close not be more reliable?